packages/addons/service/hyperion.ng/source/bin/hyperiond.start

```diff
@@ -21,14 +21,14 @@
 
 oe_setup_addon service.hyperion.ng
 
-if [ ! -f "$ADDON_HOME/hyperion.config.json" ]; then
+if [ ! -f "$ADDON_HOME/config/hyperion.config.json" ]; then
   mkdir -p $ADDON_HOME
-  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/hyperion.config.json
+  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/config/hyperion.config.json
 fi
 
-exec hyperiond $ADDON_HOME/hyperion.config.json &
+exec hyperiond --rootPath $ADDON_HOME hyperion.config.json &
 echo $! > /run/hyperiond.pid
 
 if [ -f "/usr/bin/Xorg" ]; then
   exec hyperion-x11 &
-fi
+fi
```

Comment:
Whitespace after "fi".

Reply:
mistake, will fix

Reply:
No it was actually my mistake. I thought there was an additional whitespace. Can you remove that again. Sorry for the inconveniences